### ISSUE TITLE
Quick Reply box stuck to right side

### ISSUE THREAD

**Issue by BurnyBoi** [KevinParnell/OneeChan#11]
I am seeing that regardless of which theme I choose, the Quick Reply box is stuck to the right side of the page. I can drag it up and down, but not to the left to bring it closer to the posts. I can't seem to find a setting to disable this, and it's quite inconvenient since I have a large monitor and have to bring the mouse all the way across the screen each time to reach it.

**Comment by BurnyBoi** [comment]
Just as an update, I tried messing around with the CSS, and I see why the box is stuck. It looks like it's due to the following CSS always being put into the page:

:root.right-sidebar #qr {
	right: 0 !important;
	left: auto !important;
}

When I clear out the left and right values, I am able to drag the box around as expected. I tried adding the following custom CSS to my theme:

:root.right-sidebar #qr {
	right: auto;
	left: auto;
}

Even with this, it always adds the other CSS below it, causing the box to get stuck. Sticking the box to the right side should be optional.
